content/efcore/getting-started/samples/configuring-deletion/manual-deletion-strategy/README.md

```diff
@@ -0,0 +1,12 @@
+# Manual Deletion Strategy
+
+This code example demonstrates how deletion is configured in EF Core using the OnDelete method and its parameters: Cascade Delete, SetNull, and Restrict.
```

Comment:
We say "Cascade Delete" and "SetNull". They should both either have spaces or not have spaces. If no spaces, they should be code snippets.